### DISCUSSION TITLE
Raise Governor Limit on Polygon PoS

### DISCUSSION THREAD

**Discussion by JackJMelnick** [wormhole-foundation/wormhole#3734] [Governance] [answered]
The governor limit on Wormhole for Polygon PoS network is currently set to $2M, which is low on a relative basis compared to other major EVM chains. 

With the launch of the Wormhole [native ETH bridge](https://x.com/wormholecrypto/status/1747635985091908018?s=20), transactions are beginning to get rate limited, and held up in the 24hr queue as a result of caps being hit. While this represents great news for Wormhole, as the new ETH bridge is very popular, it’s limiting the maximum capacity of Wormhole’s volume and usage growth within the Polygon ecosystem.

<img width="1498" alt="Screen Shot 2024-01-23 at 10 08 36 PM" src="https://github.com/wormhole-foundation/wormhole/assets/157543749/0371aba5-a46a-470a-a70f-5d27bd0076d4">

As a result, I’d like to propose a new governor limit to Wormhole governance for a guardian vote. After investigating similar EVM chain caps integrated with Wormhole, the proposed changes are as follows: 

Current Limit: $2,000,000 
Proposed Limit: $5,000,000 
Supporting Rationale: 
Notional USD average volume / 24 hours : $1,716,205.57 
Notional USD total volume: $53,202,372.74

**Comment by claudijd** [comment, marked as answer]
Here's some supporting data on Polygon flows to allow Guardians to make informed decisions here:

**Polygon: 30 Day**

Wormhole chain id 5 token bridge transfers over the past 31 days between sequences 188668 and 188491:

Notional usd average : $1,716,205.57
Notional usd total : $53,202,372.74

**Polygon: 6 Month**

Wormhole chain id 5 token bridge transfers over the past 181 days between sequences 188541 and 168835:

Notional usd average : $401,135.86
Notional usd total : $72,605,589.80

The Polygon current limit is set to $2M and current Governor usage over the past day is 82% of the capacity. Adjusting to $5M seems conservative step up for Guardians to consider, allowing for 3x headroom on recent usage, and based on recent flows Polygon demand appears to be on the upswing.

PR: https://github.com/wormhole-foundation/wormhole/pull/3739

**Reply by claudijd** [reply]
The above #3739 was merged, so I'll mark it as the answer.

**Comment by gbunea** [comment]
I am totally OK with this proposal from syncnode guardian perspective, as long as the market seems to need higher limit and demand seems to be increasing. Considering that all the signs show that a new bull run started we need to adjust limits so we don't negatively impact the users and the bridge can be used at maximum capacity

**Comment by dancamarg0** [comment]
Given the recent market action we'll probably have to up the limits to a few more chains soon I guess.

I'm OK with increasing the limits too (while keeping it conservative so we can adjust it slowly as needed when possible). 

They limits are there to protect the users, not to stop them.

**Comment by SimonKraus** [comment]
Staking Facilities is also supporting to raise the limits to stay operational and keep the customers satisfied.
